### PR TITLE
Change javascript and coffeescript keys in hound.yml

### PR DESCRIPTION
**Pull request by jakecraige**
Kept backwards compatibility with old naming

### CODE REVIEW

File: app/models/repo_config.rb [hunk 1]
```diff
@@ -103,4 +103,28 @@ def parse_json(content)
...
+    end
+
+    converted_config
+  end
+
+  def legacy_language_camelize(language)
+    case language.downcase
+    when "coffeescript"
+      "CoffeeScript"
+    when "javascript"
```
Comment (gylaz) [inline app/models/repo_config.rb]: Method name might be confusing since these aren't really legacy names.

**Comment by gylaz** [comment]
One comment. :+1:

File: app/models/repo_config.rb (hunk 1 above)
Comment (jakecraige) [inline app/models/repo_config.rb]: That makes sense. Fixed

**Comment by teoljungberg** [comment]
How about renaming `app/models/style_guide/java_script.rb` aswell to follow the same convention?

**Comment by salbertson** [comment]
This is great, it will help avoid issues. We should test this on staging to make sure all cases work. :+1:

**Comment by salbertson** [comment]
@jakecraige, is there anything left before this can be merged?

**Comment by jakecraige** [comment]
@salbertson I still should rename the file like @teoljungberg suggested. I can probably take care of that this afternoon

**Comment by salbertson** [comment]
Sounds good. Squash and merge when you're ready.

**Comment by jakecraige** [comment]
I actually changed my mind about renaming that file. JavaScript is the official spelling when it's capitalized. When people refer to it in lowercase it's normally "javascript". So while this change makes "javascript" acceptable. It seems to make sense that the class is still "JavaScript".

Thoughts @teoljungberg ?

**Comment by teoljungberg** [comment]
I agree, let's keep the filenames, and the classes, unchanged
---
Teo Ljungberg

**Comment by salbertson** [comment]
@jakecraige makes sense to me, merge when you're ready.

**Pull request merged**